docs/platforms/java/common/metrics/index.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: Set Up Metrics
+description: "Learn how to measure the data points you care about by configuring Metrics in your Java app."
+---
+
+<Include name="feature-stage-alpha-metrics.mdx" />
+<Note>
+
+Metrics are supported with Sentry Java SDK version `7.6.0` and above.
+
+</Note>
+
+Sentry metrics help you pinpoint and solve issues that impact user experience and app performance by measuring the data points that are important to you. You can track things like processing time, event size, user signups, and conversion rates, then correlate them back to tracing data in order to get deeper insights and solve issues faster.
+
+## Enabling metrics feature
```

Comment:
@adinauer Not sure: we should break this up for frameworks as well?

Reply:
I'm not sure what would be different if we break it up.

Reply:
I was thinking of enabling metrics via a properties file, similar to [spotlight](https://github.com/getsentry/sentry-java/blob/f4f39c79cf1314b8b737943e032934899a46c9a6/sentry-samples/sentry-samples-spring-boot-jakarta/src/main/resources/application.properties#L17).

Reply:
Ah yes please 🙏